lib/rules/i18n-named-placeholders.js

```diff
@@ -45,6 +45,11 @@ rule = module.exports = function( context ) {
 				return;
 			}
 
+			// Done if no args are passed
+			if ( node.arguments.length === 0 ) {
```

Comment:
Superbly minor: To minimize wasted effort, could get most trivial checks out of the way earlier in the function. `getCallee` is not particularly "heavy", but certainly heavier than a length check.